src/dev/build/tasks/nodejs_modules/webpack_dll.js

```diff
@@ -17,12 +17,23 @@
  * under the License.
  */
 
-import { deleteAll, read, write } from '../../lib';
-import { dirname, sep, relative } from 'path';
+import { deleteAll, isFileAccessible, read, write } from '../../lib';
+import { dirname, sep, relative, resolve } from 'path';
 import pkgUp from 'pkg-up';
 import globby from 'globby';
 
-export async function getDllEntries(manifestPath, whiteListedModules) {
+function checkDllEntryAccess(entry, baseDir = '') {
+  const resolvedPath = baseDir ? resolve(baseDir, entry) : entry;
+
+  try {
```

Comment:
This try/catch should be inside `isFileAccessible()`

Reply:
👍 changed